### PR TITLE
refactor: change format for logMessage mkString so it shows with tabs…

### PR DESCRIPTION
**Pull request by girdharshubham**
… instead of new lines which doesn't get shipped through log aggregators as a complete message and can lead to confusion

References
-

### CODE REVIEW

File: sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/DiscoveryImpl.scala [hunk 1]
```diff
@@ -189,7 +189,7 @@ class DiscoveryImpl(
     val detail = if (in.detail.isEmpty) Nil else List(in.detail)
     val seeDocs = DocLinks(sdkName).forErrorCode(in.code).map(link => s"See documentation: $link").toList
     val messages = message :: detail ::: seeDocs ::: sourceMsgs
-    val logMessage = messages.mkString("\n\n")
+    val logMessage = messages.mkString("\t|\t")
```
Comment (octonato) [inline sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/DiscoveryImpl.scala:192]: is the pipe `|` intentional?

**Review by ennru — APPROVED** [review]
LGTM.

File: sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/DiscoveryImpl.scala (hunk 1 above)
Comment (girdharshubham) [reply sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/DiscoveryImpl.scala:192]: Yes, I thought I should put a delimiter just in case.

Comment (ennru) [reply sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/DiscoveryImpl.scala:192]: It makes sense where tabs aren't rendered.

**Pull request merged**